packages/cli/templates/vue2/src/style.css

```
@@ -1,7 +1,8 @@
 @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
 
 html {
-  overflow: hidden;
+  overflow: auto;
+  hegiht: 100%;
```

```suggestion
  height: 100%;
```

some typo here @BibiSebi